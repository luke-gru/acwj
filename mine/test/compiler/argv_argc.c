int printf(char *fmt);

int main(int argc, char **argv) {
  int i;
  char *argument;
  printf("Hello world\n");

  char ****x;
  char ***p;
  int o;
  o = x + p;

  for (i=0; i < argc; i++) {
    argument= *argv;
    argv= argv + 1;
    printf("Argument %d is %s\n", i, argument);
  }
  return(0);
}
