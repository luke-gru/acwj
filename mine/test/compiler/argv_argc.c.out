Hello world
Argument 0 is ./test/compiler/out
